### PR TITLE
[HIP] Don't run failing device global test on AMD

### PR DESCRIPTION
**Pull request by hdelan**
TODO: investigate why.

### CODE REVIEW

File: sycl/test-e2e/DeviceGlobal/device_global_static.cpp [hunk 1]
```diff
@@ -3,7 +3,10 @@
 //
 // The OpenCL GPU backends do not currently support device_global backend
 // calls.
-// UNSUPPORTED: opencl && gpu
+//
+// FIXME: make this work for HIP
+//
+// UNSUPPORTED: hip_amd, opencl && gpu
```
Comment (steffenlarsen) [inline sycl/test-e2e/DeviceGlobal/device_global_static.cpp:9]:
```suggestion
// UNSUPPORTED: opencl && gpu
//
// See ADD_GH_ISSUE_LINK_HERE
// UNSUPPORTED: hip_amd
```
where `ADD_GH_ISSUE_LINK_HERE` would be the link to a Github issue detailing the failure.

Comment (hdelan) [reply sycl/test-e2e/DeviceGlobal/device_global_static.cpp]: Thanks @steffenlarsen have done so.

**Review by steffenlarsen — APPROVED** [review]
LGTM!

**Comment by hdelan** [comment]
Ping @againull . This has broken post commit CI so would be good to merge.

**Comment by sarnex** [comment]
Will merge now to unbreak HIP CI

**Pull request merged**